.github/workflows/deploy-gcp-tests.yml

```
@@ -158,17 +156,21 @@ jobs:
       #
       # SSH into the just created VM, and create a docker volume with the newly created disk.
       - name: Create ${{ inputs.test_id }} Docker volume
-        id: compute-ssh
-        uses: google-github-actions/ssh-compute@v0.1.2
-        with:
-          instance_name: ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}
-          zone: ${{ env.ZONE }}
-          ssh_private_key: ${{ secrets.GCP_SSH_PRIVATE_KEY }}
-          command: |
-            sudo mkfs.ext4 -v /dev/sdb \
-            && \
-            sudo docker volume create --driver local --opt type=ext4 --opt device=/dev/sdb \
-            ${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}
+        run: |
+          gcloud compute ssh \
+          github-service-account@${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --ssh-flag="-o ConnectionAttempts=20" \
+          --ssh-flag="-o ConnectTimeout=5" \
+          --command \
+          "\
+          sudo mkfs.ext4 -v /dev/sdb \
```

Let's check the sshd fix actually worked

```suggestion
          ps auxwww | grep sshd;
          sudo mkfs.ext4 -v /dev/sdb \
```

---

I'm not completely sure if we should leave this troubleshooting commands in the pipelines, as this just to confirm it's working the first time. The `sshd` location is standard and it's very unusual for OpenSSH to make changes to this file.